@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -4,47 +4,44 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { MathUtils } from "@here/harp-geoutils";
-import * as THREE from "three";
+import {
```

Comment:
What are we doing here? We are in Technique.ts and include sth. from Technique.ts? Luckily the compiler is not ending up in an infinite loop.

Reply:
Sorry you must've hit some bad revision i've accidentaly pushed some time ago.